Packs/Wiz/Classifiers/classifier-WizIssue.json

```diff
@@ -0,0 +1,15 @@
+{
+    "defaultIncidentType": "Wiz Issue",
```

Comment:
Yes, since we'll have more incident types soon (we'll split it from one for all to family of incident types).
Since it's already created, and going to be used soon, we added it to save the efforts later.